relay-server/src/actors/processor.rs

```diff
@@ -565,6 +573,8 @@ impl EnvelopeProcessorService {
         });
 
         let inner = InnerProcessor {
+            #[cfg(feature = "processing")]
+            redis_pool: _redis.clone().map(Arc::new),
             #[cfg(feature = "processing")]
             rate_limiter: _redis
                 .map(|pool| RedisRateLimiter::new(pool).max_limit(config.max_rate_limit())),
```

Comment:
thoughts on sending the RedisRateLimiter the same arc as in the redis_pool field in a follow-up pr?